### DISCUSSION TITLE
ナイスアイデアぼ

### DISCUSSION THREAD

**Discussion by ktakahiro150397** [ktakahiro150397/RankPrediction_Web#69] [Ideas]
誰かよろしく！

**Comment by WagashiMgmg** [comment]
#Apex募集ツイートにこのサイト乗せたら、ランクが近いプレイヤーを表示するページを表示するやつ

**Comment by ktakahiro150397** [comment]
チャート画面で実現できたらいいな一覧

## 表示データ

- 横軸ランクに対して、各列データの平均
- 横軸各列データに対して、縦軸ランク

## 表示

- ドロップダウンリストで、横軸と縦軸を自由に選択できる
- その都度、グラフ部分を選択した系列でデータの取得・表示を行う

表示のやり方は全然わかりません！
画面の部分更新が必要になるので、ajaxとか使わないとダメだと思う
